Sources/Sentry/include/SentryTransport.h

```diff
@@ -22,6 +22,12 @@ NS_SWIFT_NAME(Transport)
       attachments:(NSArray<SentryAttachment *> *)attachments
     NS_SWIFT_NAME(send(event:traceState:attachments:));
 
+- (void)sendEvent:(SentryEvent *)event
+       traceState:(nullable SentryTraceState *)traceState
+      attachments:(NSArray<SentryAttachment *> *)attachments
```

Comment:
Since this is a public API change (technically a breaking change), I wonder if we should bump a major here.
We're adding profiling support so might as well do that?

Do we need attachments here or could we get around only with the envelopeItems? Since we should easily be able to get envelopeItems from attachments assuming a SentryOptions instance is around so we can read the `maxAttachmetnSize`.